src/response/trowtoken.rs

```diff
@@ -0,0 +1,100 @@
+use failure::{Error};
+use std::io::Cursor;
+use std::time::{SystemTime, UNIX_EPOCH};
+use rocket::http::{Status};
+use rocket::http::{ContentType};
+use rocket::request::Request;
+use rocket::response::{Responder, Response};
+use crypto::sha2::Sha256;
+use chrono::Local;
+use jwt::{Header as TokenHeader, Token};
+
+const AUTHORISATION_SECRET: &str = "Bob Marley Rastafaria";
+
+#[derive(Debug, Serialize)]
+pub struct TrowToken;
+
+#[derive(Debug, Serialize, RustcEncodable, RustcDecodable)]
+pub struct BearerToken {
+    user_id: String,
+    client_id: String,
+    scope: String,
+    iat: u64,
```

Comment:
what's iat? I'm sure it's from a spec, but can we put it in a comment?

Reply:
Issued at. Thought everyone knew that! ;) Could change them to issue_time and expiry_time to make it more readable? I've been Javaing so long it's just great to use snake case!

Reply:
Yeah, I think issued_at